standalone/src/models/custom-application/custom-application-installation/builders.spec.ts

```diff
@@ -0,0 +1,30 @@
+import type { TCustomApplicationInstallationGraphql } from './types';
+import { CustomApplicationInstallationGraphql } from './index';
+
+describe('builder', () => {
+  it('GraphQL model', () => {
+    const graphqlModel =
+      CustomApplicationInstallationGraphql.random().buildGraphql<TCustomApplicationInstallationGraphql>();
+
+    expect(graphqlModel).toMatchObject({
+      __typename: 'RestrictedCustomApplicationInstallationForOrganization',
+      createdAt: expect.any(String),
+      id: expect.any(String),
+      updatedAt: expect.any(String),
+      acceptedPermissions: expect.arrayContaining([
+        expect.objectContaining({
+          __typename: 'CustomApplicationInstallationPermission',
+          name: expect.any(String),
+          oAuthScopes: expect.arrayContaining([expect.any(String)]),
+        }),
+      ]),
+      application: expect.objectContaining({
```

Comment:
Is this the model we're not asserting the `__typename` on purpose?
If so, it would be great to leave a TODO comment.

Reply:
fixed in b02bf8b02d41a325a35f359de5555c514328a6cb